src/Fpm.hs

```diff
@@ -307,3 +305,9 @@ getTestSettings [] = do
         else return []
     else return []
 getTestSettings tests = return tests
+
+makeBuildPrefix :: String -> Bool -> IO String
+makeBuildPrefix compiler release =
+  -- TODO Figure out what other info should be part of this
+  --      Probably version, and make sure to not include path to the compiler
```

Comment:
I agree, I think we will need compiler version eventually since they not compatible between versions.